apps/civil/civil-general-applications/image-repo.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: image.toolkit.fluxcd.io/v1alpha2
+kind: ImageRepository
+metadata:
+  name: civil-general-applications
+spec:
+  image: hmctspublic.azurecr.io/civil-general-applications #{"$imagepolicy": "flux-system:civil-general-applications"}
```

This is an image repo, if you add that policy it will add a tag to it, not what you are after as far as I know
```suggestion
  image: hmctspublic.azurecr.io/civil-general-applications
```